docs/user/DockerUsage.md

```diff
@@ -95,6 +97,21 @@ Any number of tasks can be run at the same time. An action can run against a sin
     viya4-deployment --tags "baseline,viya,install"
   ```
 
+- I have a new cluster, deployed in OCI using https://github.com/oracle-quickstart/oke-sas-viya-public project, and I want to install baseline and deploy viya only
+- I run as non-root user
+
+  ```bash
+  sudo docker run --rm -t \
```

Comment:
Remove the `sudo`

Reply:
I think sudo docker is legit alternative for not so savvy docker users as it ultimately worked for me instead of trying to make docker work as non-root user.